src/directives/mobilegeolocation.js

```diff
@@ -0,0 +1,199 @@
+goog.provide('ngeo.MobileGeolocationController');
+goog.provide('ngeo.mobileGeolocationDirective');
+
+goog.require('ngeo');
+goog.require('ngeo.DecorateGeolocation');
+goog.require('ngeo.FeatureOverlay');
+goog.require('ngeo.FeatureOverlayMgr');
+goog.require('ol.Feature');
+goog.require('ol.Geolocation');
+goog.require('ol.Map');
+goog.require('ol.geom.Point');
+goog.require('ol.proj');
+
+
+/**
+ * Provide a "mobile geolocation" directive.
+ *
+ * @example
+ * <ngeo-mobile-geolocation
+ *     ngeo-mobile-geolocation-map="ctrl.map"
+ *     ngeo-mobile-geolocation-options="ctrl.mobileGeolocationOptions">
+ * </ngeo-mobile-geolocation>
+ *
+ * @return {angular.Directive} The Directive Definition Object.
+ * @ngInject
+ * @ngdoc directive
+ * @ngname ngeoMobileGeolocation
+ */
+ngeo.mobileGeolocationDirective = function() {
+  return {
+    restrict: 'E',
```

Comment:
Okay, now I understand what this implies.  I'll make the fix to fit the `'A'` restriction, then.